### PR TITLE
Update pets.dmi

### PR DESCRIPTION
**Pull request by theboy6545**
## About The Pull Request

 Added some puce corgi sprites i made into pets.dmi! 

## Why It's Good For The Game

How could a puce corgi be a bad thing?. 

## Changelog
:cl:
imageadd: Added a new puce corgi. Not mapped in.
/:cl:

### CODE REVIEW

**Comment by triplezeta** [comment]
speedmerge

**Comment by Demoxide** [comment]
we need the puce corgi right now

**Comment by francinum** [comment]
Fix the changelog.

File: code/modules/mob/living/simple_animal/friendly/dog.dm [hunk 1]
```diff
@@ -585,7 +585,17 @@
 		to_chat(usr, "<span class='warning'>You can't fit this on [src]!</span>")
 		return
 	..()
-
+/mob/living/simple_animal/pet/dog/corgi/puce
```
Comment (francinum) [inline code/modules/mob/living/simple_animal/friendly/dog.dm:588]: Move this to the `NSV` folder.

Comment (francinum) [reply code/modules/mob/living/simple_animal/friendly/dog.dm:588]: Good. Now revert your changes to this file.

**Pull request closed without merging**